### PR TITLE
refactor: rename resolve_reference() from resolve_reference.py to resolve_address

### PR DESCRIPTION
**Pull request by KristianKjerstad**
## What does this pull request change?

refactor resolve_reference() name to resolve_address()
## Issues related to this change:
#465

### CODE REVIEW

File: src/common/utils/resolve_reference.py [hunk 1]
```diff
@@ -245,8 +243,12 @@ class ResolvedReference:
     entity: dict | list
 
 
-def resolve_reference(address: Address, get_data_source: Callable) -> ResolvedReference:
-    """Resolve the reference into a document."""
+def resolve_address(address: Address, get_data_source: Callable) -> ResolvedReference:
```
Comment (ingeridhellen) [inline src/common/utils/resolve_reference.py:246]: Should we also rename ResolvedReference to ResolvedAddress?

File: src/common/utils/resolve_reference.py
Comment (ingeridhellen) [inline src/common/utils/resolve_reference.py:1]: Perhaps rename the file as well?

**Review by ingeridhellen — APPROVED** [review]
Good improvement!

File: src/common/utils/resolve_reference.py (hunk 1 above)
Comment (KristianKjerstad) [reply src/common/utils/resolve_reference.py:246]: yes, that sounds good

**Pull request merged**